Unable to build 9.2.1 on Debian

Hello,
While working on packaging ATS 9.2.1 for Debian, I'm stuck with following error during tests:
```
make  test_proxy_http test_PreWarm test_HttpTransact
make[4]: Entering directory '/root/source/build/proxy/http'
/bin/bash ../../libtool  --tag=CXX   --mode=link c++ -std=c++17 -g -pipe -Wall -Wextra -Wno-ignored-qualifiers -Wno-unused-parameter -Wno-format-truncation -Wno-cast-function-type -Wno-stringop-overflow -feliminate-unused-debug-symbols -fno-strict-aliasing -Wno-invalid-offsetof -Wno-noexcept-type -Wsuggest-override -mcx16 -g -O2 -ffile-prefix-map=/root/source/build=. -fstack-protector-strong -Wformat -Werror=format-security -O3 -rdynamic -Wl,--as-needed -Wl,-z,relro -Wl,-z,now -o test_proxy_http unit_tests/test_proxy_http-unit_test_main.o unit_tests/test_proxy_http-test_ForwardedConfig.o test_proxy_http-ForwardedConfig.o unit_tests/test_proxy_http-test_error_page_selection.o test_proxy_http-HttpBodyFactory.o ../../src/tscpp/util/libtscpputil.la ../../src/tscore/libtscore.la ../../proxy/hdrs/libhdrs.a ../../iocore/eventsystem/libinkevent.a ../../proxy/logging/liblogging.a ../../lib/records/librecords_p.a ../../proxy/shared/libUglyLogStubs.a ../../mgmt/libmgmt_p.la ../../iocore/utils/libinkutils.a -lhwloc -lcap -lyaml-cpp 
libtool: link: c++ -std=c++17 -g -pipe -Wall -Wextra -Wno-ignored-qualifiers -Wno-unused-parameter -Wno-format-truncation -Wno-cast-function-type -Wno-stringop-overflow -feliminate-unused-debug-symbols -fno-strict-aliasing -Wno-invalid-offsetof -Wno-noexcept-type -Wsuggest-override -mcx16 -g -O2 -ffile-prefix-map=/root/source/build=. -fstack-protector-strong -Wformat -Werror=format-security -O3 -rdynamic -Wl,--as-needed -Wl,-z -Wl,relro -Wl,-z -Wl,now -o .libs/test_proxy_http unit_tests/test_proxy_http-unit_test_main.o unit_tests/test_proxy_http-test_ForwardedConfig.o test_proxy_http-ForwardedConfig.o unit_tests/test_proxy_http-test_error_page_selection.o test_proxy_http-HttpBodyFactory.o  ../../src/tscpp/util/.libs/libtscpputil.so ../../src/tscore/.libs/libtscore.so ../../proxy/hdrs/libhdrs.a ../../iocore/eventsystem/libinkevent.a ../../proxy/logging/liblogging.a ../../lib/records/librecords_p.a ../../proxy/shared/libUglyLogStubs.a ../../mgmt/.libs/libmgmt_p.a ../../iocore/utils/libinkutils.a -lhwloc -lcap -lyaml-cpp -Wl,-rpath -Wl,/usr/lib/trafficserver
/bin/bash ../../libtool  --tag=CXX   --mode=link c++ -std=c++17 -g -pipe -Wall -Wextra -Wno-ignored-qualifiers -Wno-unused-parameter -Wno-format-truncation -Wno-cast-function-type -Wno-stringop-overflow -feliminate-unused-debug-symbols -fno-strict-aliasing -Wno-invalid-offsetof -Wno-noexcept-type -Wsuggest-override -mcx16 -g -O2 -ffile-prefix-map=/root/source/build=. -fstack-protector-strong -Wformat -Werror=format-security -O3 -rdynamic -Wl,--as-needed -Wl,-z,relro -Wl,-z,now -o test_PreWarm unit_tests/test_PreWarm-test_PreWarm.o ../../src/tscore/libtscore.la 
libtool: link: c++ -std=c++17 -g -pipe -Wall -Wextra -Wno-ignored-qualifiers -Wno-unused-parameter -Wno-format-truncation -Wno-cast-function-type -Wno-stringop-overflow -feliminate-unused-debug-symbols -fno-strict-aliasing -Wno-invalid-offsetof -Wno-noexcept-type -Wsuggest-override -mcx16 -g -O2 -ffile-prefix-map=/root/source/build=. -fstack-protector-strong -Wformat -Werror=format-security -O3 -rdynamic -Wl,--as-needed -Wl,-z -Wl,relro -Wl,-z -Wl,now -o .libs/test_PreWarm unit_tests/test_PreWarm-test_PreWarm.o  ../../src/tscore/.libs/libtscore.so -Wl,-rpath -Wl,/usr/lib/trafficserver
/bin/bash ../../libtool  --tag=CXX   --mode=link c++ -std=c++17 -g -pipe -Wall -Wextra -Wno-ignored-qualifiers -Wno-unused-parameter -Wno-format-truncation -Wno-cast-function-type -Wno-stringop-overflow -feliminate-unused-debug-symbols -fno-strict-aliasing -Wno-invalid-offsetof -Wno-noexcept-type -Wsuggest-override -mcx16 -g -O2 -ffile-prefix-map=/root/source/build=. -fstack-protector-strong -Wformat -Werror=format-security -O3 -rdynamic -Wl,--as-needed -Wl,-z,relro -Wl,-z,now -o test_HttpTransact ../../iocore/cache/test/test_HttpTransact-stub.o unit_tests/test_HttpTransact-main.o unit_tests/test_HttpTransact-test_HttpTransact.o ../../proxy/http/libhttp.a ../../proxy/http/remap/libhttp_remap.a ../../proxy/logging/liblogging.a ../../proxy/hdrs/libhdrs.a ../../mgmt/libmgmt_p.la ../../iocore/utils/libinkutils.a ../../iocore/hostdb/libinkhostdb.a ../../iocore/dns/libinkdns.a ../../iocore/cache/libinkcache.a ../../lib/fastlz/libfastlz.a ../../iocore/aio/libinkaio.a ../../src/tscore/libtscore.la ../../src/tscpp/util/libtscpputil.la ../../proxy/libproxy.a ../../iocore/net/libinknet.a ../../lib/records/librecords_p.a ../../iocore/eventsystem/libinkevent.a -lhwloc -lpcre -lresolv -lz -llzma -lssl -lcrypto -lm 
libtool: link: c++ -std=c++17 -g -pipe -Wall -Wextra -Wno-ignored-qualifiers -Wno-unused-parameter -Wno-format-truncation -Wno-cast-function-type -Wno-stringop-overflow -feliminate-unused-debug-symbols -fno-strict-aliasing -Wno-invalid-offsetof -Wno-noexcept-type -Wsuggest-override -mcx16 -g -O2 -ffile-prefix-map=/root/source/build=. -fstack-protector-strong -Wformat -Werror=format-security -O3 -rdynamic -Wl,--as-needed -Wl,-z -Wl,relro -Wl,-z -Wl,now -o .libs/test_HttpTransact ../../iocore/cache/test/test_HttpTransact-stub.o unit_tests/test_HttpTransact-main.o unit_tests/test_HttpTransact-test_HttpTransact.o  ../../proxy/http/libhttp.a ../../proxy/http/remap/libhttp_remap.a ../../proxy/logging/liblogging.a ../../proxy/hdrs/libhdrs.a ../../mgmt/.libs/libmgmt_p.a ../../iocore/utils/libinkutils.a ../../iocore/hostdb/libinkhostdb.a ../../iocore/dns/libinkdns.a ../../iocore/cache/libinkcache.a ../../lib/fastlz/libfastlz.a ../../iocore/aio/libinkaio.a ../../src/tscore/.libs/libtscore.so ../../src/tscpp/util/.libs/libtscpputil.so ../../proxy/libproxy.a ../../iocore/net/libinknet.a ../../lib/records/librecords_p.a ../../iocore/eventsystem/libinkevent.a -lhwloc -lpcre -lresolv -lz -llzma -lssl -lcrypto -lm -Wl,-rpath -Wl,/usr/lib/trafficserver
/usr/bin/ld: ../../proxy/logging/liblogging.a(YamlLogConfig.o): undefined reference to symbol '_ZN4YAML8LoadFileERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEE'
/usr/bin/ld: /lib/x86_64-linux-gnu/libyaml-cpp.so.0.7: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
make[4]: *** [Makefile:1137: test_HttpTransact] Error 1
make[4]: Leaving directory '/root/source/build/proxy/http'
make[3]: *** [Makefile:1696: check-am] Error 2
make[3]: Leaving directory '/root/source/build/proxy/http'
make[2]: *** [Makefile:1368: check-recursive] Error 1
make[2]: Leaving directory '/root/source/build/proxy/http'
make[1]: *** [Makefile:922: check-recursive] Error 1
make[1]: Leaving directory '/root/source/build/proxy'
make: *** [Makefile:898: check-recursive] Error 1
```
I'm using yaml-cpp from Debian archive instead of the embedded one, but since they have both same version, it looks unlikely to be the root cause.
I'm more thinking about an include issue.

Any help appreciated

--- a/proxy/http/Makefile.am
+++ b/proxy/http/Makefile.am
@@ -112,14 +112,16 @@ test_proxy_http_LDADD = \
 	$(top_builddir)/iocore/utils/libinkutils.a \
 	$(top_builddir)/src/tscpp/util/libtscpputil.la \
 	@SWOC_LIBS@ @HWLOC_LIBS@ \
-	@LIBCAP@
+	@LIBCAP@ \
+	@YAMLCPP_LIBS@
 
 test_PreWarm_CPPFLAGS = \
 	$(AM_CPPFLAGS) \
 	-I$(abs_top_srcdir)/lib/catch2
 
 test_PreWarm_LDADD = \
-	$(top_builddir)/src/tscore/libtscore.la
+	$(top_builddir)/src/tscore/libtscore.la \
+	@YAMLCPP_LIBS@
 
 test_PreWarm_SOURCES = \
 	unit_tests/test_PreWarm.cc
@@ -153,7 +155,7 @@ test_HttpTransact_LDADD  =  \
 	$(top_builddir)/src/records/librecords_p.a \
 	$(top_builddir)/iocore/eventsystem/libinkevent.a \
 	-lz -llzma -lcrypto -lresolv -lssl \
-	@LIBPCRE@ @HWLOC_LIBS@ @SWOC_LIBS@
+	@LIBPCRE@ @HWLOC_LIBS@ @SWOC_LIBS@ @YAMLCPP_LIBS@
 
 test_HttpTransact_SOURCES = \
 	../../iocore/cache/test/stub.cc \

